lib/order.rb

```diff
@@ -0,0 +1,63 @@
+require './lib/menu.rb'
+require './lib/verify.rb'
+
+class Order
+
+attr_reader :current_order, :current_menu, :no_of_item, :item_total, \
+            :collect_amount, :text_message
+
+    def initialize(menu_klass: Menu, verify_klass: Verify)
+      @current_order = []
+      @current_menu = []
+      @menu_klass = menu_klass
+      @verify_klass = verify_klass
+
+    end
+
+    def start_order(item, no_of_item)
+     current_menu << menu_klass.new
+     current_order << current_menu[0].menu.select {|k,v| k == item}
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)